docs/program/apis/cloud.md

```diff
@@ -188,9 +188,281 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 {{% /tab %}}
 {{< /tabs >}}
 
+### CreateLocation
+
+Create and name a {{< glossary_tooltip term_id="location" text="location" >}} under the organization and location you are currently authenticated to.
```

Comment:
```suggestion
Create and name a {{< glossary_tooltip term_id="location" text="location" >}} under the organization you are currently authenticated to.
Optionally, make a parent location for this location.
```
Expanding intro could be good-- Something like this I think would make the parent location functionality easier to understand

Reply:
```suggestion
Create and name a {{< glossary_tooltip term_id="location" text="location" >}} under the organization you are currently authenticated to.
Optionally, put the new location under a specified parent location.
```